proto/kvrpcpb.proto

```diff
@@ -293,6 +293,12 @@ message CheckTxnStatusRequest {
     bool rollback_if_not_exist = 6;
 }
 
+enum RollbackReason {
+    NotByMe = 0;
```

Comment:
How about `AlreadyRolledBack`, since it also may because duplicated requests

Reply:
I think it's fine because "Me" can also refer to "this request".

B.T.W, RolledBack or Rollbacked, how to spell

Reply:
`NotByMe` does not look like a reason.

Reply:
Or we use 'Default' to mean no information?
When the txn status is not rollbacked, we do not need a reason, but we still have to provide a value. @coocood

Reply:
Or `NoReason` as default

Reply:
Done